calcom/cal.com
# fix: paypal setup and key validation on server

## What does this PR do?

- Adds possibility to use custom key validations per appId
- Migrates PayPal key validations to server
- Code quality refactoring
- Allows to filter viewer.integrations by appId

## Type of change




## How should this be tested?

- Install PayPal from App Store
- Input invalid keys, it should fail to create webhook
- Input valid keys, it should succeed and display a success toast

## Mandatory Tasks
